packages/dev/core/src/Shaders/default.fragment.fx

```diff
@@ -457,6 +457,14 @@ color.rgb = max(color.rgb, 0.);
             gl_FragData[PREPASS_REFLECTIVITY_INDEX] = vec4(0.0, 0.0, 0.0, writeGeometryInfo);
         #endif
     #endif
+	#ifdef PREPASS_SPECULARGLOSSINESS
+		// specularity = F0 value
+		#if defined(SPECULAR)
+			gl_FragData[PREPASS_SPECULARGLOSSINESS_INDEX] = vec4(specularMapColor.rgb * writeGeometryInfo, specularMapColor.a * writeGeometryInfo); // no specularity if no visibility

```

Comment:
It should probably be `specularColor`, which works whether the material has a spec texture or just a spec color. Reflectivity has the wrong variable too, so you can also fix it just above.

And little nit : you can just write `gl_FragData[PREPASS_SPECULARGLOSSINESS_INDEX] = vec4(specularColor, 1.0) * writegeometryInfo`

Reply:
Hi,  

Thanks for your review!
I changed the PREPASS_REFLECTIVITY and PREPASS_SPECULARGLOSSINESS to use specularColor as you suggested.

Reply:
@CraigFeldspar, would love your feedback on this one ?

Reply:
Is it failing on the PG because the mirror's spec is 1,1,1 ?

Reply:
> Is it failing on the PG because the mirror's spec is 1,1,1 ?
It seems to. In this environment, when I explicitly define its specularColor to 0, 0, 0 the rendering is goo
![PGSpecLow](https://user-images.githubusercontent.com/44207625/168648609-4b89442f-ce2a-4881-8e79-dc3639670582.png)
![PGBug](https://user-images.githubusercontent.com/44207625/168648615-8fce5fc6-ddf8-48ae-980b-2d513894f536.png)
d

Reply:
When I remove the SSR post process the problem disappears too. I guess a high reflectivity makes the SRR artifacts more visible...

Reply:
For me it makes sense that a mirror texture + a SSR post process have some artifacts. Can you fix the test so the mirror spec value is 0 ?

Reply:
to do that : change the PG url in the json file containing all tests, run the validation tests, and right click on the new image to save and replace the old validation image